### ISSUE TITLE
Keyserver and GPG fingerprint for Ubuntu installation

### ISSUE THREAD

**Issue by mendhak** [dotnet/docs#960]
# Keyserver and GPG fingerprint for Ubuntu installation
## Issues with Existing Topics

The installation instructions for [Ubuntu](https://www.microsoft.com/net/core#ubuntu) make use of the keyserver `apt-mo.trafficmanager.net` - on many corporate networks, port 11371 can be unreachable.  Since it's Ubuntu, what about using `hkp://keyserver.ubuntu.com:80` instead?  Port 80 is more often than not open, and saves many users from having to search for alternatives.  

You may also be aware of recent [GPG key ID collisions](https://www.reddit.com/r/linux/comments/4xqss3/pgp_shortid_collision_attacks_continued_now/), so while you're in there, it is probably a good time to use the GPG fingerprint instead.
## tl;dr

On the [Ubuntu installation instructions page](https://www.microsoft.com/net/core#ubuntu), change this:

```
sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
```

to

```
sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
```

**Comment by mairaw** [comment]
@blackdwarf @shanselman @neusamir this issue is regarding installation instructions on https://www.microsoft.com/net/core#ubuntu. Can one of you follow up?

**Comment by mairaw** [comment]
@blackdwarf assigning this to you but this for the .NET site, not docs

**Comment by blackdwarf** [comment]
@mairaw thanks, I belive this is being worked on.

**Comment by mairaw** [comment]
Who could confirm @blackdwarf?

**Comment by BillWagner** [comment]
@richlander @rowanmiller Can you confirm this is fixed? I looked at the page for the Ubuntu instructions, and it appears fixed.

**Comment by mairaw** [comment]
Thanks for confirming @mendhak. Closing the issue then. Page has changed since then to https://www.microsoft.com/net/core#linuxubuntu as well due to the 2.0 release.
